CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+##2015-01-21 0.8.0
+###Summary
+
+This release performs major API changes and is not backwards-compatible with
+the previous release.
```

Comment:
Yes, since this is still 0.x backwards incompat is fine, it's just best to be as explicit as possible with what's breaking in the changelog, so if you could add some information on what's backwards incompat that would be fantastic.